targets/psa_switch/externs/psa_internet_checksum.h

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2021 SYRMIA LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * Dusan Krdzic (dusan.krdzic@syrmia.com)
+ *
+ */
+
+
+#ifndef PSA_SWITCH_PSA_INTERNETCHECKSUM_H_
+#define PSA_SWITCH_PSA_INTERNETCHECKSUM_H_
+
+#include <bm/bm_sim/extern.h>
+
+namespace bm {
+
+namespace psa {
+
+class PSA_InternetChecksum : public bm::ExternType {
+ public:
+
+    BM_EXTERN_ATTRIBUTES {
+}
+
+    void init() override;
+
+    void get(Field &dst) const;
+
+    void get_verify(Field &dst, Field &equOp) const;
+
+    void clear();
+
+    void add(const std::vector<Field> fields);
+
+    void subtract(const std::vector<Field> fields);
+
+    void get_state(Field &dst) const;
+
+    void set_state(const Data &src);
+
+ private:
+    uint16_t sum;
+
+};
+
+}  // namespace bm::psa
```

Comment:
I'm surprised the style checker is accepting this, shouldn't it just be `namespace psa`?

Reply:
I did this the same as it has been done in other PSA extern implementations:
  https://github.com/p4lang/behavioral-model/blob/f71a8db11c253df6a3e663df1c0b8c887d0fab37/targets/psa_switch/externs/psa_meter.h#L62
https://github.com/p4lang/behavioral-model/blob/f71a8db11c253df6a3e663df1c0b8c887d0fab37/targets/psa_switch/externs/psa_meter.h#L62
https://github.com/p4lang/behavioral-model/blob/f71a8db11c253df6a3e663df1c0b8c887d0fab37/targets/psa_switch/externs/psa_random.h#L47

Reply:
sounds good, maybe the checker is not running on these files